assets/scss/_registry.scss

```diff
@@ -21,4 +21,10 @@
     background-color: inherit;
     border: solid 1px;
   }
+
+  // Default color attributes, in case we miss a component definition above;
+  // which has happened, see https://github.com/open-telemetry/opentelemetry.io/pull/2481.
+  $default-otel-badge-bg: $yellow;
+  color: color-contrast($default-otel-badge-bg);
+  background-color: $default-otel-badge-bg;
```

Comment:
Setting a default color is necessary for the (up coming) Bootstrap v5 (via Docsy) upgrade since, under v5, undefined badges default to white text over an unset (i.e., white) background.